app/presenters/reply_presenter.rb

```diff
@@ -19,8 +19,8 @@ def as_json(options={})
   def character(reply)
     return unless reply.character_id
     { id: reply.character_id,
-      name: reply.character_name,
-      screenname: reply.character_screenname }
+      name: reply.name,
```

Comment:
For historical record: these are overloaded in the Writable concern to properly use the value if it was preloaded from SQL, and to load the value through the associated object if it wasn't.